src/pages/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { useHistory, Switch, Route } from 'react-router-dom';
+import { DappMetaData } from 'types';
+import { importAll } from 'utils';
+import Web3 from 'web3';
+import Head from 'layout/Head';
+import Web3Wrapper from 'containers/Web3Wrapper';
+import App from './App';
+import NotFound from './NotFound';
+
+export default () => {
+  const [meta, setMeta] = React.useState<DappMetaData>({
+    tokenName: '',
+    tokenSymbol: '',
+    tokenOwnerWebsite: '',
+    siteUrl: '',
+    totalPresale: '0',
+    referralBP: '0',
+    basisPoint: '0',
+    accountCap: '0'
+  });
+
+  const [favicon, setFavicon] = React.useState<string>('');
```

Comment:
Can you directly import `useState` and use it?

Reply:
No, it will fail.